data/hai/hai ships.txt

```diff
@@ -733,6 +804,69 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 		`"Bufaer" Atomic Steering`
 		"Hyperdrive"
 
+ship "Shield Beetle Prototype"
+	sprite "ship/hai shield beetle prototype"
+	thumbnail "thumbnail/hai shield beetle"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Unfettered Militia"
+		"cost" 25400000
+		"shields" 42000
```

Comment:
This is a pretty absurd increase in shields, 10k, about a 31% increase.
Marauders of the human HWs only get about an 11% increase to their shields, when comparing to the non-marauder versions, so this is way beyond what we have so far for ships with the similar idea of revamping an existing ship type with more of some stats at the cost of others.
On that though, it doesn't seem like the extra stuff cost this ship much of anything really, it's lost some bunks and a tiny bit of cargo, the big hit being the much lowered heat dissipation, but it's getting stats like fuel capacity and weapon capacity for seemingly no reason.
The shields are what bothered me most when looking at this ship's stats, though, so a cut on what % of the Shield Beetle's shields is being added is what I reccomend. From 31% to somewhere like 20% maybe.

Reply:
this is a unique ship, it only exists once and is a prototype for you to wield, it almost has half of the heat dissipation of the other shield beetle

Reply:
"It's one of a kind" is not a justification, if anything it's an excuse, I figured it would be sold as something like that, I still take issue with the shields.

Reply:
I think a 25% increase in shields would be appropriate. The strong nerf to heat dissipation more than makes up for that, I think. The increased required crew and reduced bunks also helps balance it. 

While I haven't checked, probably the important thing is that this ship isn't available for sale. If it's a one-off ship given to the player, I'm good with the idea.

Reply:
it is doubly not available to the player : requires a license you cannot get yet, + it's not in any shipyards and will not ever be